### PR TITLE
Update webview_flutter_wkwebview to newer pigeon

### PR DESCRIPTION
**Pull request by natebosch**
The current version is incompatible with the latest `analyzer`, which is a blocker for getting the latest `mockito`, which is a blocker for getting the latest `test_api`, which is pinned by `flutter_test` and will be breaking in the next package roll.

Roll to version `7.0.5` to avoid breaking changes in newer versions.

### CODE REVIEW

File: packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFGeneratedWebKitApis.h [hunk 1]
```diff
@@ -406,8 +408,9 @@ NSObject<FlutterMessageCodec> *FWFWKWebViewConfigurationFlutterApiGetCodec(void)
 @interface FWFWKWebViewConfigurationFlutterApi : NSObject
 - (instancetype)initWithBinaryMessenger:(id<FlutterBinaryMessenger>)binaryMessenger;
 - (void)createWithIdentifier:(NSNumber *)identifier
-                  completion:(void (^)(NSError *_Nullable))completion;
```
Comment (stuartmorgan) [inline packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFGeneratedWebKitApis.h:409]: These are going to require code changes. For simplicity of unblocking your end goal, can you just update to Pigeon `^7.0.5`?

Comment (natebosch) [reply packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFGeneratedWebKitApis.h:409]: Done

**Review by stuartmorgan — APPROVED** [review]
LGTM

**Comment by stuartmorgan** [comment]
There was a const construction issue that wasn't fixed until a later breaking change, but extracting the copyright to a single file (as we do in most packages now) avoided it.

**Pull request merged**